kpatch-build/lookup.c

```diff
@@ -262,6 +283,8 @@ struct lookup_table *lookup_open(char *obj_path, char *symvers_path,
 		ERROR("malloc table");
 	memset(table, 0, sizeof(*table));
 
+	table->vmlinux = !strcmp(basename(obj_path), "vmlinux");
+
```

Comment:
It probably doesn't matter, but is it any better/worse to push this assignment down to lookup_open() just before it calls find_local_syms() ?